pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[project]
+name = "kbase-auth"
+version = "0.1.0"
+description = "Client for the KBase Authentication Service"
+readme = "README.md"
+authors = [{ name = "KBase Development Team" }]
+requires-python = ">=3.12"
+license = { text = "MIT" }
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "httpx==0.28.1",
+]
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/kbase"]
+
+[dependency-groups]
+dev = [
+    "ipython==9.5.0",
+    "pytest==8.4.2",
+    "pytest-asyncio==1.2.0",
+    "pytest-cov==7.0.0",
+    "requests==2.32.5",
+    "unasync==0.6.0",
+]
```

Comment:
I would encourage good practices by adding in a code formatting/linting section with a ruff config here. Just makes life easier not to have to worry about code format and dumb errors that can be caught by tools.

Reply:
I'm not a big fan linters at this point, they're too opinionated and catch mostlyi trivial stuff.

Reply:
It is extremely quick and easy to set up ruff to format your code -- it even looks as though PyDev has ruff as one of the formatters. I believe ruff catches trailing whitespace, and any decent IDE should also have a setting for it. You can also add a standard `.editorconfig` file to do that -- e.g. https://github.com/linkml/linkml/blob/main/.editorconfig

We have already spent more time debating this than it would have taken to add a section to the pyproject.toml file with the setup for formatting and linting.

Reply:
I have had the opposite experience.

Reply:
Well, it seems we're at an impasse - can you agree to leave things as they are for now and add a linter when / if it becomes an issue?